website/content/commands/connect/envoy.mdx

```diff
@@ -227,6 +231,29 @@ This example assumes that the correct [environment variables](#api-options) are
 used to set the local agent connection information and ACL token, or that the
 agent is using all-default configuration.
 
+### Sidecar Proxy with ACLs Enabled
+
+In secure deployments, Consul's ACL system is enabled with a default `deny` policy.
+Nothing can access Consul API resources unless it presents a Consul ACL token with the
+necessary privileges. A sidecar proxy must have a Consul ACL token to act as itself and
+the service it fronts, and to access all that service's potential upstreams.
+The best way to generate such a token is to
+[use a service identity](/consul/commands/acl/token/create#create-a-token-for-a-service).
+
+Then, provide that token when launching the Envoy sidecar proxy instance with
+the `consul connect envoy` command through either:
+- Environment variables
+  [`CONSUL_HTTP_TOKEN`](/consul/commands#consul_http_token) or
+  [`CONSUL_HTTP_TOKEN_FILE`](/consul/commands#consul_http_token_file)
+- CLI flags
+  [`-token`](/consul/commands/connect/envoy#token) or
+  [`-token-file`](/consul/commands/connect/envoy#token-file)
+
+```shell-session
+$ export CONSUL_HTTP_TOKEN="<secret-id-of-token-with-service-identity-for-web>"
+$ consul connect envoy -sidecar-for web
+```
```

Comment:
```suggestion
After establishing a service identity that provides access to resources, you can provide the token when launching the Envoy sidecar proxy instance with
the `consul connect envoy` command. You can pass the token in an environment variable or with the `-token` or `-token-file` flag.

Environment variables:    
- [`CONSUL_HTTP_TOKEN`](/consul/commands#consul_http_token)     
- [`CONSUL_HTTP_TOKEN_FILE`](/consul/commands#consul_http_token_file)

CLI flags:
- [`-token`](/consul/commands/connect/envoy#token) 
- [`-token-file`](/consul/commands/connect/envoy#token-file)

```shell-session
$ export CONSUL_HTTP_TOKEN="<secret-id-of-token-with-service-identity-for-web>"
$ consul connect envoy -sidecar-for web
```
```
This suggestion is about presenting information in complete sentences, rather than completing the sentences with bullets.